src/MessageList/MessageListItem.vala

```diff
@@ -68,10 +68,23 @@ public class Mail.MessageListItem : Gtk.ListBoxRow {
         loading_cancellable = new GLib.Cancellable ();
 
         style_context = get_style_context ();
-        style_context.add_class ("card");
+        style_context.add_class (Granite.STYLE_CLASS_CARD);
 
-        var avatar = new Granite.Widgets.Avatar.with_default_icon (48);
-        avatar.valign = Gtk.Align.START;
+        string parsed_address;
+        string parsed_name;
+
+        var camel_address = new Camel.InternetAddress ();
+        camel_address.unformat (message_info.from);
+        camel_address.ref ();
```

Comment:
You're leaking a reference here
```suggestion
```

Reply:
Without the ref I get a crash from double free